test/acceptance/voronoi.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+var assert = require('assert');
+
+var Analysis = require('../../lib/analysis');
+
+var testConfig = require('../test-config');
+var QueryRunner = require('../../lib/postgresql/query-runner');
+
+describe('voronoi analysis', function() {
+
+    var queryRunner;
+
+    before(function() {
+        queryRunner = new QueryRunner(testConfig.db);
+    });
+
+    var QUERY = 'select * from atm_machines';
+
+    var sourceAtmMachines = {
+        type: 'source',
+        params: {
+            query: QUERY
+        }
+    };
+
+    var config = testConfig.create({
+        batch: {
+            inlineExecution: true
+        }
+    });
+
+    function performAnalysis(definition, callback) {
```

Comment:
BTW there is now a test-helper that allows you to create analyses and run queries: https://github.com/CartoDB/camshaft/blob/69a1657c34bf4a621ba9ef05f32c93464358514b/test/acceptance/lazy-nodes.js#L73-L88